bin/build_image.sh

```diff
@@ -43,10 +43,10 @@ fi
 if [ -e chef/${TEMPLATE_NAME}/Berksfile ] ; then
   export BERKSHELF_PATH="chef/berkshelf"
   rm -rf $BERKSHELF_PATH chef/${TEMPLATE_NAME}/cookbooks chef/${TEMPLATE_NAME}/Berksfile.lock
-  berks vendor -b chef/${TEMPLATE_NAME}/Berksfile chef/${TEMPLATE_NAME}/cookbooks
+  berks vendor --delete -b chef/${TEMPLATE_NAME}/Berksfile chef/${TEMPLATE_NAME}/cookbooks
 fi
 export PACKER_LOG=1
-packer build -force -debug $(basename $TEMPLATE)
+packer build -color=false -force -debug $(basename $TEMPLATE)
```

Comment:
Are we planning on keeping the `-debug` switch?

Reply:
Yes. That would provide additional info necessary for any debugging. Also, Jenkins client seems to handle the requirement for pressing enter on every step when prompted, so it is not detrimental.

Ideally, I would want a single debug flag passable to a build so that debug can be turned on all step but that is a feature so I won't work on it right now.